examples/test-cra/.storybook/config.js

```diff
@@ -1,9 +1,10 @@
 import { configure } from '@storybook/react';
 import { setOptions } from '@storybook/addon-options';
 
-setOptions({
-  sidebarAnimations: false,
-});
+setTimeout(
```

Comment:
unsure if actually needed, but is consistent with react-native, and functions correctly.